client/src/components/related/RelatedProductsModule.jsx

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import RelatedProductList from './lists/RelatedProductList.jsx';
+import YourOutfitList from './lists/YourOutfitList.jsx';
+import api from '../../../../client/src/lib/api.js';
 
 const RelatedProductsModule = () => {
+  const [relatedItems, setRelatedItems] = useState(['no related products']);
+  const [outfitList, setOutfitList] = useState([]);
+
+  const getProductList = (err, result) => {
```

Comment:
considering you're not using err or results in here, and have error handling via .catch rather than error first callbacks, you might consider rewriting this to take in no arguments. This function is simple enough it's not really a problem, but it can be confusing when you declare parameters that aren't needed or used.

Reply:
Good catch! I removed them and will double check for more.